### PR TITLE
feat: internationalization Bulgarian

### PR DESCRIPTION
**Pull request by dmushkov**
Provides login, messages, and console in Bulgarian.
Adds missing language references to fr, it and zh where other languages are used.

- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: console/src/app/app.component.ts [hunk 1]
```diff
@@ -268,15 +267,15 @@ export class AppComponent implements OnDestroy {
   }
 
   private setLanguage(): void {
-    this.translate.addLangs(supportedLanguages);
-    this.translate.setDefaultLang(fallbackLanguage);
+    this.translate.addLangs(['de', 'en', 'es', 'fr', 'it', 'ja', 'pl', 'zh']);
```
Comment (peintnermax) [inline console/src/app/app.component.ts:270]: can you remerge main branch and add it to supportedLanguages? Thx

File: console/src/app/app.component.ts [hunk 2]
```diff
@@ -268,15 +267,15 @@ export class AppComponent implements OnDestroy {
...
+    this.translate.setDefaultLang('en');
 
     this.authService.user.subscribe((userprofile) => {
       if (userprofile) {
-        const cropped = navigator.language.split('-')[0] ?? fallbackLanguage;
-        const fallbackLang = cropped.match(supportedLanguagesRegexp) ? cropped : fallbackLanguage;
+        const cropped = navigator.language.split('-')[0] ?? 'en';
+        const fallbackLang = cropped.match(/de|en|es|fr|it|ja|pl|zh/) ? cropped : 'en';
```
Comment (peintnermax) [inline console/src/app/app.component.ts:276]: same here and pls add `bg` to supported Languages

**Review by peintnermax — CHANGES_REQUESTED** [review]
Thx for your great work and your contribution. We have made some changes recently so can you please remerge main. Rest looks good to me 👍

**Review by eliobischof — APPROVED** [review]

**Review by peintnermax — APPROVED** [review]
Thanks a lot for your contribution @dmushkov 🎉 

We appreciate your work and if you're interested we would send you some swag. Just send us a mail to [hi@zitadel.com](mailto:hi@zitadel.com). We will then send you a form with some questions about the shirt size and other details 👍

**Pull request merged**